.env

```diff
@@ -1,6 +1,7 @@
 VUE_APP_TYPEKIT_ID="egx1cmg"
+VUE_APP_GA_TRACKING_ID="UA-115750203-1"
```

Comment:
I think the value needs to go to `.env.production`. In this file we should set it to nothing. Otherwise we will be tracking ourselves during development.

Reply:
So in this file the line will be `VUE_APP_GA_TRACKING_ID=`